raiden-ts/src/transfers/epics/locked.ts

```diff
@@ -131,8 +131,14 @@ function makeAndSignTransfer$(
     'expiration too soon',
   );
 
+  // fee is added to the lock amount
+  const amount_with_fee = decode(
```

Comment:
Please, use `camelCase` in our codebase. `snake_case` is allowed only because and on external APIs and messages which already use it.